client/src/components/ProductOverview/ProductOverview.scss

```diff
@@ -243,13 +243,14 @@ $standardBorder: 1px solid $standardColor;
     display: flex;
     justify-content: center;
 
+    #product-description {
+      border-right: $standardBorder;
+      border-width: 2px;
```

Comment:
Consider not using a hard coded pixel, remember Leslie said about hard coding, maybe 0.5 ems?